cmake/configs/posix_rpi_common.cmake

```diff
@@ -74,6 +74,7 @@ set(config_module_list
 	#
 	# PX4 drivers
 	#
+	drivers/linux_sbus
```

Comment:
That's something you have to add to the Devguide please - https://github.com/PX4/Devguide. It doesn't make sense to push this to the flightstack without properly document it.